src/grid-list/grid-tile.jsx

```diff
@@ -113,74 +163,18 @@ const GridTile = React.createClass({
     this._ensureImageCover();
   },
 
-  //to update theme inside state whenever a new theme is passed down
-  //from the parent / owner using context
   componentWillReceiveProps(nextProps, nextContext) {
-    let newMuiTheme = nextContext.muiTheme ? nextContext.muiTheme : this.state.muiTheme;
-    this.setState({muiTheme: newMuiTheme});
+    this.setState({
+      muiTheme: nextContext.muiTheme || this.state.muiTheme,
+    });
   },
 
-  getStyles() {
-    const spacing = this.state.muiTheme.rawTheme.spacing;
-    const themeVariables = this.state.muiTheme.gridTile;
-    const actionPos = this.props.actionIcon ? this.props.actionPosition : null;
-    const gutterLess = spacing.desktopGutterLess;
-
-    let styles = {
-      root: {
-        position: 'relative',
-        display: 'block',
-        height: '100%',
-        overflow: 'hidden',
-      },
-      titleBar: {
-        position: 'absolute',
-        left: 0,
-        right: 0,
-        [this.props.titlePosition]: 0,
-        height: this.props.subtitle ? 68 : 48,
-        background: this.props.titleBackground,
-        display: 'flex',
-        alignItems: 'center',
-      },
-      titleWrap: {
-        flexGrow: 1,
-        marginLeft: actionPos !== 'left' ? gutterLess : 0,
-        marginRight: actionPos === 'left' ? gutterLess : 0,
-        color: themeVariables.textColor,
-        overflow: 'hidden',
-      },
-      title: {
-        fontSize: '16px',
-        textOverflow: 'ellipsis',
-        overflow: 'hidden',
-        whiteSpace: 'nowrap',
-      },
-      subtitle: {
-        fontSize: '12px',
-        textOverflow: 'ellipsis',
-        overflow: 'hidden',
-        whiteSpace: 'nowrap',
-      },
-      actionIcon: {
-        order: actionPos === 'left' ? -1 : 1,
-      },
-      childImg: {
-        height: '100%',
-        transform: 'translateX(-50%)',
-        position: 'relative',
-        left: '50%',
-      },
-    };
-    return styles;
-  },
-
-  componeneDidUpdate() {
+  componentDidUpdate() {
```

Comment:
I noticed the `componentDidUpdate` method was spelled wrong as `componeneDidUpdate`. Unless React's lifecycle methods support spelling checking, I'm guessing this wasn't doing anything :laughing: 

I named it back, which obviously is changing the behavior. Though I'm thinking that the behavior is probably right to do that stuff on both `mount` and `update`.